client/src/components/employer-component/employer-info-component/index.jsx

```diff
@@ -0,0 +1,172 @@
+import { HeaderContainer, TopRowContainer } from "./index.styles";
+import EmployerType from "../../../prop-types/EmployerType";
+import UserType from "../../../prop-types/UserType";
+
+import UserChipComponent from "../../shared/user-chip-component";
+import ChangeOwnerDialog from "../../shared/change-owner-dialog";
+import ErrorScreenComponent from "../../shared/error-screen-component";
+
+function EmployerInfoComponent({
+  employer,
+  getUserById,
+  managedUsers,
+  setSnackBarMessage,
+}) {
+  const vertButtonRef = useRef(null);
+  const [anchorEl, setAnchorEl] = useState(null);
+  const open = Boolean(anchorEl);
+
+  let owner = getUserById(employer.owner);
+  let creator = getUserById(employer.creator);
+
+  useEffect(() => {
+    owner = getUserById(employer.owner);
+    creator = getUserById(employer.creator);
+  }, [employer]);
+
+  const handleClick = () => {
+    setAnchorEl(vertButtonRef.current);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const [ownerChangeDialog, setOwnerChangeDialog] = React.useState(false);
+  const [error, setError] = React.useState(null);
+
+  const onEditOwnerClick = () => {
+    setOwnerChangeDialog(true);
+  };
+
+  const onOwnerChangeConfirm = () => {
+    setOwnerChangeDialog(false);
+  };
+
+  const onChangeOwnerCancel = () => {
+    setOwnerChangeDialog(false);
+  };
+
+  if (error) return <ErrorScreenComponent message={error} />;
+
+  return (
+    <TopRowContainer
+      style={{ display: "flex", justifyContent: "space-between" }}
+    >
+      <div>
+        <HeaderContainer>
+          <Typography
+            style={{
+              fontFamily: "Arial",
+              fontSize: "34px",
+              fontWeight: 700,
+              lineHeight: "42px",
+              letterSpacing: "0.25px",
+              textAlign: "left",
+            }}
+          >
+            {employer.name}
+          </Typography>
+          <IconButton onClick={handleClick}>
+            <MoreVertIcon
+              sx={{
+                color: "gray",
+                cursor: "pointer",
+              }}
+              ref={vertButtonRef}
+            />
+          </IconButton>
+
+          <Menu
+            anchorEl={anchorEl}
+            open={open}
+            onClose={handleClose}
+            MenuListProps={{
+              "aria-labelledby": "split-button",
+            }}
+            PaperProps={{
+              style: {
+                width: vertButtonRef ? 190 : null,
+              },
+            }}
+          >
+            <MenuItem onClick={handleClose}>
+              <IconButton>
+                <DownloadIcon />
+              </IconButton>{" "}
+              Export as JSON
+            </MenuItem>
+            <MenuItem onClick={handleClose}>
+              <IconButton>
+                <DeleteIcon />
+              </IconButton>
+              Delete
+            </MenuItem>
+          </Menu>
+        </HeaderContainer>
+        <Typography
```

Comment:
Take a look at edit-job-lead-header. We can have this style matching the typography for Job Listing

```
<Typography
          style={{
            fontFamily: "Arial",
            fontSize: "14px",
            fontWeight: 400,
            lineHeight: "24px",
            letterSpacing: "0.15px",
            textAlign: "left",
          }}
        >
          Job Listing
        </Typography>
```